index.js

```diff
@@ -1 +1,95 @@
-module.exports = require('./eslintrc.json')
+const fs = require('fs')
+const path = require('path')
+
+/**
+ * Based on the location of this file (assumed to be node_modules/eslint-config-frost-standard/index.js)
+ * we want to read in the package.json file from the root of the project, and return the project name
+ * @returns {String} the package name
+ */
+function getProjectName () {
+  const pkgJson = JSON.parse(fs.readFileSync(path.join(__dirname, '..', '..', 'package.json'), {encoding: 'utf8'}))
+  return pkgJson.name
+}
+
+const localPrefixes = [
+  '../',
+  './',
+  'dummy/'
+]
+
+try {
+  const projectName = getProjectName()
+  localPrefixes.push(projectName)
```

Comment:
@juwara0 @notmessenger This will prevent consumers from having to re-define the `ocd/sort-import-declarations` rule within their own `.eslintrc.js`.